src/components/data/listing/Listing.js

```diff
@@ -47,6 +47,11 @@ import {
     INFO_TYPES_QUERY_KEY,
 } from "serviceFacades/filesystem";
 
+import {
+    getDefaultsMapping,
+    DEFAULTS_MAPPING_QUERY_KEY,
+} from "serviceFacades/instantlaunches";
```

Comment:
I put this here because I wasn't sure that this needed to be a context, and I read that contexts should be used sparingly.